pkgs/development/libraries/pipewire/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p nix-update -i bash
+# shellcheck shell=bash
+
+set -o errexit -o pipefail -o nounset -o errtrace
+shopt -s inherit_errexit
+shopt -s nullglob
+IFS=$'\n'
+
+NIXPKGS_ROOT="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")/../../../.."
```

Comment:
Just a suggestion
```suggestion
NIXPKGS_ROOT="$(git rev-parse --show-toplevel)"
```
This assumes that this is run in a Git checkout of nixpkgs but that is most likely the case.

Reply:
Ah, that's true. Thanks!